src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -236,14 +236,32 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
-
+            string script = @"
+function __Check-Help {
```

Comment:
Works for me. It's another instance of copying from elsewhere (Expand-Alias). But just because it's in use elsewhere doesn't mean we should keep doing it that way 😄